src/mnemonics/electrum-words.cpp

```diff
@@ -209,22 +218,22 @@ namespace
    * \param unique_prefix_length  the prefix length of each word to use for checksum
    * \return                      True if the test passed false if not.
    */
-  bool checksum_test(std::vector<std::string> seed, uint32_t unique_prefix_length)
+  bool checksum_test(std::vector<epee::wipeable_string> seed, uint32_t unique_prefix_length)
```

Comment:
Is `seed` being modified? I think this can be `const &`.

Reply:
It is (CRC removed)